jupyterlab-dataops/Dockerfile

```diff
@@ -40,6 +36,21 @@ COPY resources/jupyter/ /etc/jupyter/
 # Expose port for Vertex AI compatibility
 EXPOSE 8080
 
-FROM base as bigquery
+# use default cloudbeaver in non-bigquery target
+FROM base AS default-stage
+COPY --from=dbeaver/cloudbeaver:22.0.5 /opt/cloudbeaver/ /opt/cloudbeaver/
+COPY resources/cloudbeaver/conf /opt/cloudbeaver/conf
+
+FROM base as bigquery-stage
```

Comment:
This step should have the same name that is stored inside this var
```
target: ${{ matrix.provider }}
```
For example 

Our strategy should look like that:
```
    strategy:
      matrix:
        provider: ["bigquery-stage"]
```
If we want to apply this target while building docker image

Reply:
It is needed to add some prefix/suffix to stage in Dockerfile. You can't simply parametrize this with `FROM ${PROVIDER} as final`. The change should be to pass passing matrix argument as `build_arg` rather than `target`. I missed that.